packages/manager/apps/catalog-revamp/src/components/Filters/Filters.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState } from 'react';
+import { OsdsButton } from '@ovhcloud/ods-components/button/react';
+import { OsdsText } from '@ovhcloud/ods-components/text/react';
+import { OsdsLink } from '@ovhcloud/ods-components/link/react';
+import { ODS_TEXT_LEVEL, ODS_TEXT_SIZE } from '@ovhcloud/ods-components/text';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { useTranslation } from 'react-i18next';
+import FilterItem from './FilterItem';
+import './Filters.scss';
+import {
+  getAvailableCategoriesWithCounter,
+  getUniverses,
+  Product,
+} from '@/utils/utils';
+
+interface FiltersProps {
+  products: Product[];
+  setSelectedCategories: React.Dispatch<React.SetStateAction<string[]>>;
+  setSelectedUniverses: React.Dispatch<React.SetStateAction<string[]>>;
+  onApply: () => void;
+}
+
+const Filters: React.FC<FiltersProps> = React.memo(
```

Comment:
Why use React.memo ?

Reply:
React.memo is used to wrap the entire component, which means that React will skip rendering the component and reuse the last rendered result if the props have not changed. This can improve performance.

Reply:
You didn't understand the question : I didn't ask "What does React.memo do ?" I asked why you used it here ?

Reply:
<img width="389" alt="image" src="https://github.com/ovh/manager/assets/33665069/c2d4603b-9bbf-4d1f-88c2-7410e8a0f550">
Because on the Profiling tool (React Developer Tools) I have seen that with useMemo it was faster if I apply for example 4 categories, in that scenario I was able to load the component in 5ms instead of 9ms
<img width="389" alt="image" src="https://github.com/ovh/manager/assets/33665069/b5f02e2b-6a25-4ca0-b5e8-f2ee1516deea">
useMemo avoid to rerender and with the props (categories and universes) in that case.
What is your opinion about it ?